### PR TITLE
build: simplify Android build

### PR DESCRIPTION
**Pull request by junhochoi**
We only need Android NDK to build quiche for Android platform.
Simply use rust image from Travis and install NDK.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -117,31 +107,19 @@ matrix:
...
      env:
        NDK_VER_OLD=r13b
        NDK_VER=r21
-       CMAKE_VER=3.6.4111459
-     android:
-       components:
-        - build-tools-26.0.1
-        # Minimum API level supported
-        - android-21
+       NDK_URL=https://dl.google.com/android/repository/android-ndk-%s-linux-x86_64.zip
```
Comment (ghedo) [inline .travis.yml:115]: I think it's better to leave this in the `script` section, so it doesn't show up in the build configuration https://travis-ci.com/github/cloudflare/quiche/jobs/361909618 which is not very useful.

Comment (junhochoi) [reply .travis.yml]: Done.

Comment (junhochoi) [reply .travis.yml]: And rebased.

**Comment by junhochoi** [comment]
It saved 30-40sec in android build time in Travis.

**Pull request merged**